nwb_conversion_tools/datainterfaces/ecephys/neuralynx/neuralynxdatainterface.py

```diff
@@ -0,0 +1,35 @@
+"""Authors: Cody Baker."""
+import numpy as np
+from pathlib import Path
+from natsort import natsorted
+from typing import Union
+
+from spikeextractors import MultiRecordingChannelExtractor, NeuralynxRecordingExtractor
+
+
+from ..baserecordingextractorinterface import BaseRecordingExtractorInterface
+from ....utils.json_schema import get_schema_from_method_signature
+
+PathType = Union[str, Path]
+
+
+class NeuralynxRecordingInterface(BaseRecordingExtractorInterface):
+    """Primary data interface class for converting the Neuralynx format."""
+
+    RX = MultiRecordingChannelExtractor
+
+    @classmethod
+    def get_source_schema(cls):
+        """Compile input schema for the RecordingExtractor."""
+        return get_schema_from_method_signature(cls.__init__)
```

Comment:
until we automate path detection, we'll need to add that folder_path is of format "directory"